packages/botonic-plugin-contentful/src/tools/l10n/csv-import.ts

```diff
@@ -1,22 +1,20 @@
 import { ManageCms } from '../../manage-cms/manage-cms'
 import * as cms from '../../cms'
-import {
-  BotonicContentType,
-  CMS,
-  CmsException,
-  ContentId,
-  ContentType,
-} from '../../cms'
+import { CMS, ContentId, ContentType } from '../../cms'
```

Comment:
Multiple imports from `../../cm`. Maybe `CMS`, `ContentId` and `ContentType` are the only ones needed.

Reply:
@elozano98 done in future PR after refactor. thanks!